FueledUtils/LoadingState.swift

```diff
@@ -38,7 +58,22 @@ public enum LoadingState<Error: Swift.Error> {
 }
 
 public extension Action {
-	public func loadingState() -> SignalProducer<LoadingState<Error>, NoError> {
+	///
+	/// **Deprecated**: Please use `getSafely(at:)` instead.
+	///
+	/// Refer to the documentation for `getSafely(at:)` for more info.
+	///
+	@available(*, deprecated, renamed: "loadingState")
+	// The unused parameter allows to bypass the compiler error "Invalid redeclaration of 'loadingState'",
+	// while retaining backward compatibility
+	public func loadingState(_ unused: Void = ()) -> SignalProducer<LoadingState<Error>, NoError> {
```

Comment:
@notbenoit I had to do this unfortunately to make sure the compiler didn't complain, and it seems to work in practice so I'm leaving it as is